pennylane/hf/integrals.py

```diff
@@ -308,6 +308,207 @@ def overlap_integral(*args):
     return overlap_integral
 
 
+def _hermite_moment(alpha, beta, t, e, rc):
```

Comment:
flyby comment to add: this is such a well documented function, it seems a shame to keep it private?

Reply:
This is a good point Josh. We have all such functions, which are very unlikely to be used by users, as private ones but I will think about having them public.

Reply:
Right, but remember that developers are users as well :)

Reply:
Sure, I make it public.